src/drivers/intel/hda/hda-dma.c

```diff
@@ -570,10 +569,6 @@ static int hda_dma_release(struct dma_chan_data *channel)
 	 */
 	hda_chan->state |= HDA_STATE_RELEASE;
 
-	if (channel->direction == DMA_DIR_HMEM_TO_LMEM ||
-	    channel->direction == DMA_DIR_LMEM_TO_HMEM)
-		ret = hda_dma_host_start(channel);
-
```

Comment:
Same, not able to follow this simplification - not sure why we did a host_start in a dma_release in the first place.

Reply:
yeah, this change looks unrelated to RELEASE removal ? Line 570 also sets the RELEASE state, should we remove RELEASE completely ?